src/protocol/tx_validity.md

```diff
@@ -147,7 +147,7 @@ for input in tx.inputs:
         if tx.witnesses[input.witnessIndex].dataLength != 64:
             return False
         # Signature must be from owner
-        if address_from(ecrecover(txhash(), tx.witnesses[input.witnessIndex].data)) != input.owner:
+        if address_from(eckrecover(txhash(), tx.witnesses[input.witnessIndex].data)) != input.owner:
```

Comment:
I did it to match the convention of the opcodes to distinguish on the curve it is recovering. However, since this is the same as ethereum's ecrecover, I can revert this change if the original is clear enough.

Reply:
Wouldn't the following make a bit more sense `ecrecover_r1` and `ecrecover_k1`?

Reply:
Bumping this to get an answer

Reply:
agreed, will make this change